sql/2020/03_Markup/summary_pages_by_device.sql

```diff
@@ -0,0 +1,25 @@
+#standardSQL
+# summary_pages data grouped by device
+
+# live is very cheap
+
+CREATE TEMP FUNCTION AS_PERCENT (freq FLOAT64, total FLOAT64) RETURNS FLOAT64 AS (
+  ROUND(SAFE_DIVIDE(freq, total), 4)
+);
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  COUNTIF(TRIM(doctype) <> '') AS freq_doctype,
+  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
+  AS_PERCENT(COUNTIF(TRIM(doctype) <> ''), SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX)) AS pct_doctype_m102, 
+  MIN(bytesHtml) AS min_bytes_html_m109, 
+  MAX(bytesHtml) AS max_bytes_html_m108, 
+  ROUND(AVG(bytesHtml),0) AS avg_bytes_html_m110, 
```

Comment:
Should these be percentiles?

Reply:
Actually, I see this is it's own standalone query. Can these be removed?

Reply:
These are to cover specific requests made by the team. I added bytesHtml as a percentile already as I thought that may be of more use.